components/logging/org.wso2.carbon.logging.view/src/main/java/org/wso2/carbon/logging/view/appenders/LoggingAppender.java

```diff
@@ -38,6 +38,16 @@
 public class LoggingAppender implements PaxAppender {
 
     private CircularBuffer<LogEvent> circularBuffer;
+    private static String ip;
+
+    static {
+        try {
+            InetAddress localHost = InetAddress.getLocalHost();
+            ip = localHost.getHostAddress();
+        } catch (UnknownHostException var3) {
+            ip = "127.0.0.1";
```

Comment:
Is it possible to put this string to a constant file?

Reply:
I think we can keep this as it is since there isn't a constant file that is accessible from this file.